copter/source/docs/systemid-mode-operation.rst

```diff
@@ -95,3 +95,142 @@ main loop frequency divided by a sub-sample factor.
 | 0            |    0         | 8             |
 +--------------+--------------+---------------+
 
+Identification of a Multicopter
+===============================
+Model Scope
+-------------------------------
+
+The system and its components, that are included in the mathmatical model to be identified is shown in the following block diagram. It covers the motor mixer, the 
+motors and ESCs as well as the airframe of the copter, the gyroscopes and their lowpass filters. 
+The frequency sweep inputs 10-12 are used for the identification, since they provide a direct input to the regarded system. Because of this, the input signal is not 
+directly influenced by the controllers compared to the other chirp inputs, thereby making it independent from the parameterization of the controller.
+
+.. image:: ../images/ControlSystemDiagram.png
+:name: fig-ctrl-sys-ardupilot
+
+Model Structure
+------------------------------
+
+.. image:: ../images/bodyDiagramQuadcopter.PNG
```

Comment:
@fbredeme The positive sign of the rotation about the bx and by axes is not as I would expect and is counter to the aircraft reference systems that I have seen in texts.  Using the right hand rule, positive rotation about the bx axis would be roll to the right and positive rotation about the by axis would be pitch up.  This is also opposite to the positive rotations for the IMU data.  Any reason that you chose the positive directions like this?

Reply:
@bnsgeyer No, I just adopted it incorrectly from the literatur. Thanks for the correction.